sql/core/benchmarks/ExtractBenchmark-jdk11-results.txt

```diff
@@ -2,118 +2,137 @@ Java HotSpot(TM) 64-Bit Server VM 11.0.5+10-LTS on Mac OS X 10.15.3
+ISOYEAR of timestamp                                927            938          12         10.8          92.7       0.3X
+QUARTER of timestamp                                959            963           6         10.4          95.9       0.3X
+MONTH of timestamp                                  852            864          18         11.7          85.2       0.4X
+WEEK of timestamp                                  1124           1252         112          8.9         112.4       0.3X
+DAY of timestamp                                    848            867          19         11.8          84.8       0.4X
+DAYOFWEEK of timestamp                              977            987          16         10.2          97.7       0.3X
+DOW of timestamp                                    945            964          18         10.6          94.5       0.3X
+ISODOW of timestamp                                 924            929           5         10.8          92.4       0.3X
+DOY of timestamp                                    852            906          67         11.7          85.2       0.4X
+HOUR of timestamp                                   665            671           5         15.0          66.5       0.5X
+MINUTE of timestamp                                 655            670          15         15.3          65.5       0.5X
+SECOND of timestamp                                 757            763           7         13.2          75.7       0.4X
+MILLISECONDS of timestamp                           745            761          14         13.4          74.5       0.4X
+MICROSECONDS of timestamp                           691            697           7         14.5          69.1       0.5X
+EPOCH of timestamp                                  794            806          12         12.6          79.4       0.4X
 
 Java HotSpot(TM) 64-Bit Server VM 11.0.5+10-LTS on Mac OS X 10.15.3
 Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz
 Invoke date_part for timestamp:           Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-cast to timestamp                                   303            323          24         33.0          30.3       1.0X
-MILLENNIUM of timestamp                             841            852           9         11.9          84.1       0.4X
-CENTURY of timestamp                                818            821           4         12.2          81.8       0.4X
-DECADE of timestamp                                 821            836          15         12.2          82.1       0.4X
-YEAR of timestamp                                   858            928          61         11.7          85.8       0.4X
-ISOYEAR of timestamp                                933            939           6         10.7          93.3       0.3X
```

Comment:
Any ideas why the numbers became worse. Did you run the benchmark in the same environment?

Reply:
```
[info] ISOYEAR of timestamp                                949           1031          73         10.5          94.9       0.2X
[info] QUARTER of timestamp                               1065           1097          39          9.4         106.5       0.2X
```
with a new round result, I guess the results are random